allensdk/brain_observatory/ecephys/copy_utility/__main__.py

```diff
@@ -9,7 +9,7 @@
 
 from allensdk.config.manifest import Manifest
 from ._schemas import InputSchema, OutputSchema, available_hashers
-from ..argschema_utilities import write_or_print_outputs
+from allensdk.brain_observatory.nwb.argschema_utilities import write_or_print_outputs
```

Comment:
Why put argschema utilities in nwb rather than brain observatory. It is used in non-nwb ecephys modules (this one, for instance).

Reply:
Got it. Thanks, change coming...